front/cypress/e2e/survey_builder/survey_question_logic.cy.ts

```diff
@@ -22,7 +22,7 @@ describe('Survey question logic', () => {
 
   it('allows setting logic for select question', () => {
     cy.setAdminLoginCookie();
-    cy.intercept('GET', `/web_api/v1/admin/phases/${phaseId}/custom_fields`).as(
+    cy.intercept('GET', `/web_api/v1/phases/${phaseId}/custom_fields`).as(
```

Comment:
Always important to run the e2e tests when making route changes. I thought we did this but I guess it was missed somehow.